resources/texts/change_log.txt

```diff
@@ -17,15 +17,20 @@ When printing with a 100% infill the infill pattern will change to ZigzZag for a
 We’ve streamlined the user login authentication by removing any restrictions, especially for strict enterprise-level IT requirements. 
 
 * Other new features and improvements:
-- TPU top layers 
+- Improved TPU: top layers have large bridge distance 
 - Add warning icon to show which extruder is causing the configuration to be 'Not Supported', contributed by fieldOfView
 - Show what's new pages with every Cura build
 - Speed up loading of settings list
 - Re-use vertex buffer objects in rendering
 - Add Build Volume Temperature value to ChangeAtZ, contributed by legend069
 - Allow plugins to have multiple views, contributed by Tyronnosaurus
+- Reduced top/bottom speed for TPU
+- Increased lined width for 0.3mm layer height profiles
 
 * Bug fixes:
+- Fixed a crash when installing plugins. 
```

Comment:
We didn't! We just added more logging so that we could track why it's happening.